src/main/java/com/savvato/tribeapp/services/ToBeReviewedCheckerServiceImpl.java

```diff
@@ -111,11 +111,7 @@ public void updateTables(ToBeReviewed tbr, boolean hasMatchingRejectedPhrase, bo
             toBeReviewedRepository.setHasBeenGroomedTrue(tbr.getId());
```

Comment:
`toBeReviewedRepository.setHasBeenGroomedTrue(tbr.getId());` is called by line 96. Since that is the only place it is called, we can replace line 96 with 111, and then remove the `if (phraseValid)` check on 110. 

In the ticket, when I say the `first case`, this is it.